### PR TITLE
Updated North Sámi translations in the cookie files

### PR DESCRIPTION
**Pull request by nikopartanen**
We noticed there were some missing Sámi translations in the cookie setting section, and those are now all checked and translated.

### CODE REVIEW

File: languages/CookieConsent/se.ini [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Accept All Cookies = "Dohkket buot niesttážiid"
```
Comment (EreMaijala) [inline languages/CookieConsent/se.ini]: This file should be in the upstream vufind version. You can add these with Lokalise or with a pull request to vufind-org/vufind.

Comment (EreMaijala) [reply languages/CookieConsent/se.ini]: Or I can create the pull request on your behalf if you wish so.

**Review by EreMaijala — CHANGES_REQUESTED** [review]

File: languages/CookieConsent/se.ini (hunk 1 above)
Comment (nikopartanen) [reply languages/CookieConsent/se.ini]: I added this now, Lokalise seems to be working very nicely

**Comment by nikopartanen** [comment]
Should I delete that line from the North Sámi translation file?

**Comment by EreMaijala** [comment]
As discussed in Slack, everything in languages needs to go upstream, so Lokalise. You'll need to remove the whole file from this pull request.

**Comment by nikopartanen** [comment]
I added now the content of that file into Lokalize and removed the file in GitHub within this pull request, please let me know if anything else should be corrected.

**Review by EreMaijala — APPROVED** [review]

**Pull request merged**